crates/sui/src/main.rs

```diff
@@ -53,21 +53,21 @@ async fn main() {
                 .init()
         }
         SuiCommand::Client {
-            cmd: Some(ReplayTransaction { gas_info, .. }),
+            cmd: Some(ReplayTransaction {
+                gas_info, ptb_info, ..
+            }),
             ..
         } => {
+            let mut config = telemetry_subscribers::TelemetryConfig::new()
```

Comment:
@laura-makdah, I've seen this piece of code change quite a bit across a couple of PRs from you -- could you describe what the rationale for the changes are, and why it keeps changing?  This pattern of introducing lots of different trace targets is new to me.

Reply:
From what I understand, the issue is that the `replay` command has a bunch of `info` traces. This was changed to `info` for all commands, which made the CLI table output to be mixed with `info` traces. Then I changed it to `error`, which also changed the `replay` part without realizing that `replay` needs `info`.. There was another PR that changed back to `error` for the `replay` command only.

Reply:
I followed up in person, but for those following along here the extra info commands are currently contained fully in the replay crate to avoid dependency injection of the verbose flags. The trace tags act as a switch to add different output lines.